Minecraft crashes When opening Energy Core GUI

### Nomi CEu Version

1.7-beta-6a

### Launcher

Curseforge , also in HMCL

### Configurations or Mods Changed

Added AE2 Fluid Crafting Rework AE2-2.6.4-r
Added Flux Networks 1.12.2-4.1.1.34

### Environment

Singleplayer

### Mode

Normal Mode

### What Happened

While Opening Energy Core GUI for Draconic Evolution, Minecraft no response and crashes
![image](https://github.com/user-attachments/assets/49a9b370-0501-4e2e-88f8-756be572617f)


### Expected Behavior

no crash

### Reproduction Steps

Open Energy Core GUI

### Logs

https://mclo.gs/PZNGmA4

And
```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// There are four lights!

Time: 2024-08-22 18:15:46 CST
Description: Error executing task

java.util.concurrent.ExecutionException: io.netty.util.IllegalReferenceCountException: refCnt: 0
    at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    at java.util.concurrent.FutureTask.get(FutureTask.java:192)
    at net.minecraft.util.Util.runTask(SourceFile:531)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:1817)
    at java.lang.Thread.run(Thread.java:750)
Caused by: io.netty.util.IllegalReferenceCountException: refCnt: 0
    at io.netty.buffer.AbstractByteBuf.ensureAccessible(AbstractByteBuf.java:1408)
    at io.netty.buffer.AbstractByteBuf.checkIndex(AbstractByteBuf.java:1347)
    at io.netty.buffer.UnpooledHeapByteBuf.copy(UnpooledHeapByteBuf.java:536)
    at io.netty.buffer.AbstractByteBuf.copy(AbstractByteBuf.java:1163)
    at codechicken.lib.packet.PacketCustom.copy(PacketCustom.java:773)
    at codechicken.lib.packet.PacketCustom.toPacket(PacketCustom.java:320)
    at codechicken.lib.packet.PacketCustom.sendToPlayer(PacketCustom.java:359)
    at com.brandon3055.brandonscore.lib.datamanager.TileDataManager.lambda$detectAndSendChangesToListeners$1(TileDataManager.java:90)
    at com.brandon3055.brandonscore.utils.DataUtils.forEachMatch(DataUtils.java:101)
    at com.brandon3055.brandonscore.lib.datamanager.TileDataManager.detectAndSendChangesToListeners(TileDataManager.java:90)
    at com.brandon3055.brandonscore.inventory.ContainerBCBase.detectAndSendChanges(ContainerBCBase.java:61)
    at net.minecraft.inventory.Container.addListener(Container.java:53)
    at com.brandon3055.brandonscore.inventory.ContainerBCBase.addListener(ContainerBCBase.java:66)
    at net.minecraftforge.fml.common.network.internal.FMLNetworkHandler.openGui(FMLNetworkHandler.java:101)
    at com.brandon3055.draconicevolution.blocks.tileentity.TileEnergyStorageCore.onStructureClicked(TileEnergyStorageCore.java:151)
    at com.brandon3055.draconicevolution.blocks.tileentity.TileInvisECoreBlock.onTileClicked(TileInvisECoreBlock.java:72)
    at com.brandon3055.draconicevolution.blocks.InvisECoreBlock.onBlockActivated(InvisECoreBlock.java:73)
    at net.minecraft.server.management.PlayerInteractionManager.processRightClickBlock(PlayerInteractionManager.java:449)
    at net.minecraft.network.NetHandlerPlayServer.processTryUseItemOnBlock(NetHandlerPlayServer.java:741)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:55)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:11)
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at net.minecraft.util.Util.runTask(SourceFile:529)
    ... 5 more

No Mixin Metadata is found in the Stacktrace.
```

### Additional Information

The save was created in 1.7-alpha-4

--- a/overrides/config/Universal Tweaks - Mod Integration.cfg
+++ b/overrides/config/Universal Tweaks - Mod Integration.cfg
@@ -496,7 +496,7 @@ general {
 
     "codechicken lib" {
         # Fixes network ByteBuf leaks from PacketCustom
-        B:"Packet Leak Fix"=true
+        B:"Packet Leak Fix"=false
     }
 
     "ender storage" {

